### DISCUSSION TITLE
GG

### DISCUSSION THREAD

**Discussion by obkduishinijf** [3kh0/ext-remover#536] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
still waiting for an alternative method of disabling GG, point blank works but lags the chromebook (since they're trash.) and proxies work just fine until teachers start their class. pretty much everything else doesn't work, downgrading is risky, so i've got no idea what to do. anyone know any alternate methods of disabling, or preferrably killing it?

**Comment by funkymokey** [comment]
just downgrade, how is it even risky though

**Reply by obkduishinijf** [reply]
school constantly does daily checks on the chromebooks to see if anything changed. apparently some kid downgraded and got banned from chromebooks.

**Reply by funkymokey** [reply]
why cant you just say that you dont update your chromebook

**Reply by bartholemuew** [reply]
> why cant you just say that you dont update your chromebook

chromebooks autoupdate

**Reply by funkymokey** [reply]
> > why cant you just say that you dont update your chromebook
> 
> chromebooks autoupdate

what about caub? is it suspicious to them also?

**Reply by obkduishinijf** [reply]
> > > why cant you just say that you dont update your chromebook
> > 
> > 
> > chromebooks autoupdate
> 
> what about caub? is it suspicious to them also?

they quite literally check every single corner of the computer, i don't have any freedom but at least point blank works, just laggy.
